src/main/java/seedu/opus/logic/commands/SortCommand.java

```diff
@@ -22,6 +25,11 @@ public SortCommand(String keyword) {
     @Override
     public CommandResult execute() {
         model.sortList(keyword);
-        return new CommandResult(MESSAGE_SUCCESS);
+        if (keyword.contains("all") || keyword.contains("status") || keyword.contains("priority")
+                || keyword.contains("start") || keyword.contains("end")) {
```

Comment:
IMO it might be cleaner to put the commands into a static array called `SORTABLE_BY` and loop through it to check.